src/plays/countries-statics/Country.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from "react";
+
+export default function Country({ activeGeo }) {
+  const [country, setCountry] = useState([]);
+  const [isloading, setIsLoading] = useState(false);
+  const getCountry = () => {
+    setIsLoading(true);
+    fetch(`https://restcountries.com/v3.1/alpha/${activeGeo}`)
+      .then((response) => response.json())
+      .then((data) => {
+        setIsLoading(false);
+        setCountry(data);
+      })
+      .catch((error) => console.log(error));
```

Comment:
You should take care of the loading in error too.

Also, show the error message on UI, not just on the console.